### ISSUE TITLE
N1 becomes unusable and throws error about empty keymap.json file when launching after new update

### ISSUE THREAD

**Issue by osakkers** [nylas/N1#2223]
When I launch Nylas N1, directly after installing the latest update, this happens:

<img width="793" alt="nylas" src="https://cloud.githubusercontent.com/assets/19175074/15275438/76d90bc8-1ad4-11e6-8c57-5edb838e5730.png">

Not sure what the issue is, but I can't use it.

***

What operating system are you using?
El Capitan 

What version of N1 are you using?
Latest: 0.4.37 (5/15/16)
It was working great before the update.

**Comment by osakkers** [comment]
![image](https://cloud.githubusercontent.com/assets/19175074/15275521/cacf84bc-1ad6-11e6-9d34-ec78c3ca5ba3.png)

**Comment by michaelmarlow** [comment]
Same thing here.

```
index.js:28 SyntaxError: Unexpected end of input
    at Object.parse (native)
    at KeymapFile.load.e [as load] (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\keymap-manager.js:51:24)
    at KeymapManager.loadKeymaps (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\keymap-manager.js:108:23)
    at NylasEnvConstructor.module.exports.NylasEnvConstructor.startWindow (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\nylas-env.js:703:20)
    at NylasEnvConstructor.module.exports.NylasEnvConstructor.startSecondaryWindow (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\nylas-env.js:749:12)
    at Object.<anonymous> (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\secondary-window-bootstrap.js:20:12)
    at Object.<anonymous> (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\secondary-window-bootstrap.js:31:4)
    at Module._compile (module.js:413:34)
    at Object.defineProperty.value [as .js] (C:\Users\mike\AppData\Local\nylas\app-0.4.36\resources\app.asar\src\compile-cache.js:204:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at setupWindow (file:///C:/Users/mike/AppData/Local/nylas/app-0.4.36/resources/app.asar/static/index.js:70:3)
    at window.onload (file:///C:/Users/mike/AppData/Local/nylas/app-0.4.36/resources/app.asar/static/index.js:109:5)
```

**Comment by bengotow** [comment]
Hey! Thanks for reporting this - do you have a `~/.nylas/keymap.json` file? Can you post the contents of it? I haven't been able to reproduce this just yet, but it definitely shouldn't be hanging.

**Comment by michaelmarlow** [comment]
I did have a keymap.json file with empty contents. I deleted it, restarted N1, and the problem has been resolved.

**Comment by bengotow** [comment]
Thanks - this should be fixed in 7d5ecacb14281bfbb39bfc1fa73c9191dc6ac3a6 and we'll ship a patch to fix it.

**Comment by bengotow** [comment]
Hey! This has shipped in 0.4.37. If you were impacted by this problem, you can grab the update directly from Releases (https://github.com/nylas/N1/releases).

**Comment by jstejada** [comment]
Hey folks, v0.4.40 has shipped with a fix and should be hitting the auto updater soon. Feel free to reopen or submit a new issue if you continue having trouble. Thanks!
